horreum-backend/pom.xml

```diff
@@ -179,6 +179,16 @@
     </dependencies>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Plugin version 3.2.0 is already set in parent POM.

Reply:
Good I can remove this.